src/main/resources/reference.conf

```diff
@@ -24,15 +24,14 @@ nautilus.metering {
   }
 
   agent {
-    socket-path: "/tmp/metering-agent-ipc.sock"
-    ipc-timeout: 10 seconds
-    ipc-backlog: 2048
+    host: "0.0.0.0"
+    port: 3050
   }
 
   api {
     host: "0.0.0.0"
     port: 8080
-    keys: [ "" ]
+    keys: [ "test" ]
```

Comment:
I think we can start using single key and and make all of those fields configurable by the env variables

Reply:
This is a reference file meant to get the local development environment up. For any other environment, the expectation is that an external configuration is mapped in via file mount.